### ISSUE TITLE
Chaos

### ISSUE THREAD

**Issue by psociety** [sollaholla/unity-simple-tp-controller#5]
I think it would be a good idea to move all non essential third party assets to a subfolder like:
`/Assets/Vendor/`

Or if all packages are required, under:
`/Assets/SimpleTPController/Vendor/`

So you just need to copy a single folder/ it's more clear when using your package on another project :)

And i would also list the usage keys as there is no hint when joining the scene :)

And i don't know if it's the expected behaviour, but i see no icons when pressing TAB key:
![image](https://user-images.githubusercontent.com/29523682/82429975-fc775780-9a8c-11ea-87e5-f1f412b872ba.png)

**Comment by sollaholla** [comment]
Yeah, lol. Sadly this project is pretty much dead for the time being. Also yeah the icons were not yet integrated at this point in development.

**Comment by psociety** [comment]
Oh, what a shame, you sure spent big time making the kinematic implementation and inventory system.
¡Thank you for creating it and making it open source!
